torch/ao/quantization/fx/_decomposed.py

```diff
@@ -27,6 +27,40 @@ def _dequantize_per_tensor_impl(
 ) -> torch.Tensor:
     return (input.to(torch.float32) - zero_point) * scale
 
+def _quantize_per_channel_impl(
+        input: torch.Tensor,
+        scales: torch.Tensor,
+        zero_points: torch.Tensor,
+        axis: int,
```

Comment:
is `dim` here better instead of `axis` to match better pytorch conventions?

Reply:
yeah probably, maybe we can rename this later since this is used in existing quantize_per_channel ops right now as well